schema/listener/listener.go

```diff
@@ -1,7 +1,9 @@
-package indexerbase
+package listener
```

Comment:
In the spirit of minimizing the need for extra go modules I've made this a sub-package just called `listener`. Does this seem okay within the context of `schema`? (meaning this is basically `schema/listener` - it's a little weird but maybe okay)

Reply:
What if we called this package `blockdata` and the main type would be `blockdata.Listener`? Seems a bit more logical to have `schema/blockdata.Listener` than `schema/listener.Listener`